<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="tab_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="info_tabs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="main_tab">
        <attribute name="title">
         <string>Main</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="ms_per_step_label">
           <property name="text">
            <string>Time Per Step (ms)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="ms_per_step_spinner">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="real_time_factor_label">
           <property name="text">
            <string>RealTime Factor</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="real_time_factor_spinner">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="maze_tab">
        <attribute name="title">
         <string>Maze</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="load_maze_button">
           <property name="text">
            <string>Load Maze File</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="edit_baze_button">
           <property name="text">
            <string>Edit Maze</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="random_maze_button">
           <property name="text">
            <string>Random Maze</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="maze_file_name_label">
           <property name="text">
            <string>No Maze</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="editing_label">
           <property name="text">
            <string>Not Editing</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="right_side_layout">
        <item>
         <widget class="QTabWidget" name="gui_tabs">
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="tabsClosable">
           <bool>false</bool>
          </property>
          <property name="movable">
           <bool>false</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="time_status_layout">
          <item>
           <widget class="QPushButton" name="step_button">
            <property name="text">
             <string>Step</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="step_spinner">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="play_button">
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pause_button">
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="real_time_label">
            <property name="text">
             <string>RTF:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="real_time_value_label">
            <property name="text">
             <string>1.0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="time_label">
            <property name="text">
             <string>Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="time_value_label">
            <property name="text">
             <string>00 00:00:00.000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionSourceCode"/>
    <addaction name="actionWiki"/>
   </widget>
   <widget class="QMenu" name="menuWorld">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionReset_Mouse"/>
    <addaction name="actionReset_Time"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuWorld"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSourceCode">
   <property name="text">
    <string>Source Code</string>
   </property>
  </action>
  <action name="actionDebug_Console">
   <property name="text">
    <string>Debug Console</string>
   </property>
  </action>
  <action name="actionReset_Mouse">
   <property name="text">
    <string>Reset Mouse</string>
   </property>
  </action>
  <action name="actionReset_Time">
   <property name="text">
    <string>Reset Time</string>
   </property>
  </action>
  <action name="actionWiki">
   <property name="text">
    <string>Wiki</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../simulator.qrc"/>
 </resources>
 <connections/>
</ui>
